components/Layout.js

```diff
@@ -0,0 +1,5 @@
+import Navigation from "@/components/Navigation";
+
+export default function Layout() {
+  return <Navigation />;
+}
```

Comment:
Wir haben hier noch die {children} property drin, weil wir das Layout in der app-Component um den Rest gewrapt haben. Bestimmt geht es auch so, wie Ihr es macht. Ich weiß nicht, was besser ist. Wir haben uns an die Vorlage in dem Lights-Beispiel gehalten.